### DISCUSSION TITLE
Firestore doc changes type

### DISCUSSION THREAD

**Discussion by jongbonga** [capawesome-team/capacitor-firebase#466] [Q&A] [answered]
Hi @robingenz thanks again for the plugins and congratulations on the release of 5.2.0

Is there any way of getting the [change type](https://firebase.google.com/docs/firestore/query-data/listen#view_changes_between_snapshots) from collection snapshots?

**Comment by robingenz** [comment, marked as answer]
Hi @jongbonga,
No, this is not yet supported. Feel free to create a feature request and I will make sure it is included in the next version.
